x-pack/plugins/security_solution/public/common/components/filter_group/filter_by_assignees.tsx

```diff
@@ -51,19 +56,31 @@ export const FilterByAssigneesPopover: FC<FilterByAssigneesPopoverProps> = memo(
           assignedUserIds={assignedUserIds}
           showUnassignedOption={true}
           button={
-            <EuiFilterButton
-              data-test-subj={TEST_IDS.FILTER_BY_ASSIGNEES_BUTTON}
-              iconType="arrowDown"
-              badgeColor="subdued"
-              onClick={togglePopover}
-              isSelected={isPopoverOpen}
-              hasActiveFilters={selectedAssignees.length > 0}
-              numActiveFilters={selectedAssignees.length}
+            <EuiToolTip
+              position="bottom"
+              content={
+                !isPlatinumPlus && upsellingMessage
+                  ? upsellingMessage
+                  : i18n.translate('xpack.securitySolution.filtersGroup.assignees.popoverTooltip', {
+                      defaultMessage: 'Filter by assignee',
+                    })
+              }
```

Comment:
The `useUpsellingMessage` hook will only return the message if it needs to be displayed (the license requirement defined [here](https://github.com/elastic/kibana/pull/172014/files#diff-bd02811362a8caa59d65ac906bf16c759bf62916d58d1a063255634c8cbad45bR115) is not met), so there's no need to do the license check again:

```suggestion
              content={
                upsellingMessage ?? i18n.translate('xpack.securitySolution.filtersGroup.assignees.popoverTooltip', {
                  defaultMessage: 'Filter by assignee',
                })
              }
```